src/uportQRDisplay.js

```diff
@@ -1,26 +0,0 @@
-import { Uport } from './uport'
-import { QRUtil } from './util/qrdisplay'
-
-
-class UportQRDisplay extends Uport {
-  constructor(dappName, opts = {}) {
-    super(dappName, opts)
-    this.openQR = QRUtil.openQr
-    this.closeQR = QRUtil.closeQr
-  }
-
-  request({uri, topic}) {
-    // TODO need self reference
-    console.log(this)
-    this.openQR(uri)
-    return new Promise((resolve, reject) => {
-      this.msgServer.waitForResult(topic, (err, res) => {
-        this.closeQR()
-        if (err) reject(err)
-        resolve(res)
-      })
-    })
-  }
-}
-
-export { UportQRDisplay }
```

Comment:
We can experiment with various solutions. It should be fairly easy to do. Through the build process you can change QRutils to be an {} which would eliminate that whole line of code for example.

No need for us to worry about it, this week. But it's fairly loosely coupled right now.